content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -21,14 +21,6 @@ This page shows how to enable and configure encryption of secret data at rest.
 
 {{% /capture %}}
```

Comment:
While you're at it, can you check on the status of encryption at rest? (It's possible line 20 is no longer relevant)